go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/grafana/alerting-api v0.0.0-20210311171115-b0eb4577f38c
 	github.com/grafana/grafana-aws-sdk v0.2.0
 	github.com/grafana/grafana-plugin-model v0.0.0-20190930120109-1fc953a61fb4
-	github.com/grafana/grafana-plugin-sdk-go v0.88.0
+	github.com/grafana/grafana-plugin-sdk-go v0.88.1-0.20210316010403-be8d0c26d32a
```

Comment:
This includes the FrameToJSON function... we should use a tagged release before merging!